region/brinstar/kraid.json

```diff
@@ -824,6 +824,12 @@
           "nodeSubType": "hidden",
           "nodeItem": "Missile",
           "nodeAddress": "0x789EC"
+        },
+        {
+          "id": 5,
+          "name": "KiHunter Junction (Kihunters Killed)",
```

Comment:
I don't think I like calling this kihunters killed, as there are strats to go through without killing them. Making it so this is no longer free makes it so there is some concern about keeping track of if they are dead, i.e. going from the left/bottom to the item and back. Maybe the kihunters should be an obstacle instead

Reply:
See this is the part I didn't explain - you can safely kill the kihunters from the center hole or the right side morph tunnel.  That leaves XRay climb and damage boost as strats where you could take another hit from not killing the Kis.

Reply:
An obstacle instead of a node may be a better way to show this.

Reply:
The obstacle isn't doing much, but it should make things clearer